addons/website/static/src/scss/website.scss

```diff
@@ -232,6 +232,7 @@ font[class*='bg-'] {
 .oe_img_bg {
     background-size: cover;
     background-repeat: no-repeat;
+    background-position: 50% 50%;
```

Comment:
I'm not sure about this due to compatibility but I imagine the user will be able to fix it easily